wdio/specs/homepage.js

```diff
@@ -117,4 +117,25 @@ describe('Municipality homepage', () => {
     const target = browser.getAttribute('.item.action.homepage-teaser:nth-child(4) .content .header a', 'target');
     assert(target == '_blank');
   });
+
+  it('should show QAs only for the chosen user type "residents" and the hebrew language', () => {
+    browser.url('/municipality-1/node/1?user_type=residents&language=he');
+    browser.waitForVisible('div=כמה סייעות יש בגן ילדים?');
+    browser.waitForVisible('div=באילו שעות מפנים את הזבל?');
+  });
+
+  it('should not show QAs for "businesses" when "residents" are the chosen user type ', () => {
+    assert(!browser.isVisible('div= האם גני ילדים זכאים לפטור מארנונה?'));
```

Comment:
I think we should get all the FAQs inside the box for example and then loop through them and make sure we are getting what we need, maybe this is not possible with WDIO but the important thing right now is to make the tests pass with the new design, then we can check if we can improve this.